native/golang/types/common/common.go

```diff
@@ -89,6 +89,8 @@ type UssItem struct {
 	Name string `json:"name"`
 	// Whether the item is a directory
 	IsDir bool `json:"isDir"`
+	// The permission string of the item
+	Mode string `json:"mode"`
```

Comment:
Dumb question, could we also get the file tag? 🙏

Reply:
I think its possible, but may not be very straightforward depending on the z/OS version. AFAIK, z/OS 3.1 returns this info in a newer function in the C stdlib, but this does not exist on previous versions and you have to call a service instead.

I'll track it in a new GitHub issue 👍